Assets/BidMachine/Api/CustomExtras.cs

```diff
@@ -0,0 +1,17 @@
+using System;
+using System.Collections.Generic;
+
+namespace BidMachineAds.Unity.Api
+{
+    [Serializable]
+    public sealed class CustomExtras
```

Comment:
I’ve added a new `CustomExtras` model that mirrors the behaviour of the `CustomParams` model. The decision to use a separate model rather than CustomParams is to anticipate potential future differences between these two.